backend/drm/drm.c

```diff
@@ -586,24 +573,22 @@ static bool wlr_drm_connector_set_cursor(struct wlr_output *output,
 
 	wlr_drm_surface_make_current(&plane->surf);
 
-	wlr_texture_upload_pixels(plane->wlr_tex, WL_SHM_FORMAT_ARGB8888,
+	struct wlr_tex *tex = wlr_tex_from_pixels(rend, WL_SHM_FORMAT_ARGB8888,
 		stride, width, height, buf);
```

Comment:
It looks pretty easy to pass a `wlr_tex` to `wlr_drm_connector_set_cursor` for the DRM backend. Do you think that would still work when multi-GPU is implemented? In other words: can this `rend` render _any_ `wlr_tex`, no matter where it comes from?